src/pages/promo.scss

```diff
@@ -0,0 +1,54 @@
+@import '../shared/styles/mixins';
+
+.page-promo {
+  padding: 90px 0;
+
+  .discuss-button {
+    @include animated-button;
+    box-sizing: border-box;
+    width: auto;
+    padding: 0 15px;
+    margin-top: 70px;
+  }
+
+  .promo-title {
+    @include block-header;
+  }
+
+  .devdo-list {
+    display: grid;
+    grid-row-gap: 105px;
+    margin-top: 45px;
+  }
+
+  .devdo-block-text a {
```

Comment:
and here

Reply:
Description for this block presume any level of nesting ( `>` may not work), and `className` may not be defined due to JSON source. So `.devdo-block-text a` is more reliable solution in this case